.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,36 @@
+name: Node.js CI
```

Comment:
please use the latest version of the workflow from https://github.com/mljs/.github/blob/master/workflow-templates/nodejs.yml